ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4375,4 +4375,65 @@ EC2SC:
   - supply_air_temperature_sensor
   - supply_air_temperature_setpoint
   implements:
-  - CONTROL
\ No newline at end of file
+  - CONTROL
+
+CHWISVPCM:
+  description: "Chllled water isolation valve percentage control and monitoring"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - chilled_water_flowrate_alarm
+  - chilled_water_isolation_valve_mode
+  - chilled_water_isolation_valve_percentage_setpoint
+  uses:
+  - chilled_water_isolation_valve_percentage_command
+  - chilled_water_isolation_valve_percentage_sensor
+
+CDWISVPCM:
+  description: "Condensing water isolation valve percentage control and monitoring"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - condensing_water_flowrate_alarm
+  - condensing_water_isolation_valve_mode
+  - condensing_water_isolation_valve_percentage_setpoint
```

Comment:
Yes, it is different from that type because the condensing_water_isolation_valve_percentage_setpoint is one where user input while the condensing_water_isolation_valve_percentage_command is the feedback on the user input.

Reply:
see above, please change.

Reply:
the setpoint has been removed